test/unit/dimensions.js

```diff
@@ -29,21 +29,28 @@ function fn( val ) {
 */
 
 function testWidth( val, assert ) {
-	assert.expect( 9 );
+	assert.expect( 14 );
 	var $div, $empty;
 
+	function testElement( $elem ) {
+		$elem.width( val( 30 ) );
+		var nodeName = $elem[ 0 ].nodeName;
+		assert.equal( $elem.width(), 30, nodeName + ": Test set to 30 correctly" );
+		$elem.css( "display", "none" );
+		assert.equal( $elem.width(), 30, nodeName + ": Test hidden element" );
+		$elem.css( "display", "" );
+		$elem.width( val( -1 ) ); // handle negative numbers by setting to 0 #11604
+		assert.equal( $elem.width(), 0, nodeName + ": Test negative width normalized to 0" );
+		$elem.css( "padding", "20px" );
+		assert.equal( $elem.width(), 0, nodeName + ": Test padding specified with pixels" );
+		$elem.css( "border", "2px solid #fff" );
+		assert.equal( $elem.width(), 0, nodeName + ": Test border specified with pixels" );
+	}
+
 	$div = jQuery( "#nothiddendiv" );
-	$div.width( val( 30 ) );
-	assert.equal( $div.width(), 30, "Test set to 30 correctly" );
-	$div.css( "display", "none" );
-	assert.equal( $div.width(), 30, "Test hidden div" );
-	$div.css( "display", "" );
-	$div.width( val( -1 ) ); // handle negative numbers by setting to 0 #11604
-	assert.equal( $div.width(), 0, "Test negative width normalized to 0" );
-	$div.css( "padding", "20px" );
-	assert.equal( $div.width(), 0, "Test padding specified with pixels" );
-	$div.css( "border", "2px solid #fff" );
-	assert.equal( $div.width(), 0, "Test border specified with pixels" );
+	testElement( $div );
+	// Test span/inline element
+	testElement( jQuery( "#display" ) );
```

Comment:
If it gets removed the test will fail but if it got changed from a span to a div, it wouldn't, would it?